worlds/fe8/data/chapter_unit_blocks.json

```diff
@@ -272,7 +276,7 @@
             "name": "LArachelGroup",
             "base": 9149644,
             "count": 2,
-            "logic": {"must_fight": {"at_least": 1}}
```

Comment:
Can we try to add a `must_fight: at_least 2` to L'arachel/Dozla/Rennac's initial spawn?

Reply:
if by official spawn you mean her group in chapter 4 it should already be there?